arch/arc/soc/quark_se_c1000_ss/dts.fixup

```diff
@@ -68,4 +68,10 @@
 #define CONFIG_GPIO_QMSI_1_IRQ		INTEL_QMSI_GPIO_B0800B00_IRQ_0
 #define CONFIG_GPIO_QMSI_1_IRQ_PRI	INTEL_QMSI_GPIO_B0800B00_IRQ_0_PRIORITY
 
+#define CONFIG_ADC_0_IRQ		INTEL_QMSI_ADC_80015000_IRQ_NORMAL
```

Comment:
we are moving away from QMSI, so why are you using INTEL_QMSI here? this is generic designware IP!

Reply:
I will change the name to DW_ADC_80015000_IRQ_NORMAL. Hope its fine